lib/bright/sis_apis/focus.rb

```diff
@@ -66,7 +66,7 @@ def get_students(params = {}, options = {})
             :total => total_results,
             :per_page => params[:limit],
             :load_more_call => load_more_call,
-            :no_threads => options[:no_threads]
+            :no_threads => 2
```

Comment:
Let's define this at the top maybe as a constant and then we can do something more like this in case we want to pass the option in later:

```suggestion
            :no_threads => options[:no_threads] || DEFAULT_NO_THREADS
```